requirements.yml

```diff
@@ -1 +0,0 @@
-edpm_ansible/requirements.yml
```

Comment:
Since you have replaced the symlink with the the file itself, care to delete the one in edpm_ansible folder?

Reply:
You are right, I've deleted it, thx.